### DISCUSSION TITLE
How to transfer file from iphone to papermerge?

### DISCUSSION THREAD

**Discussion by grbharati** [ciur/papermerge#441] [Q&A]
Hello all:

I am just a beginner and trying to figure out application in iOS that can scan and send the scanned file directly to a papermerge folder.
There are loads of scanner applications in the store, but I am not sure what feature I should be looking to accomplish my need. How are you all sending files from phone to papermerge directly?

Thank you, and happy holidays!

**Comment by ciur** [comment]
At this point it is not straightforward how to do import documents from external sources. Upcoming version is fully RESTful - which means other people will be able to build solutions/apps/scripts to import documents into Papermerge from various sources, in very particular case from iPhone app into a Papermerge instance.

**Comment by grbharati** [comment]
Right now, I am using scanner pro with webdav. And Papermerge has access to webdav folder.
